kmod/pfckmod.c

```diff
@@ -56,6 +58,16 @@ static ssize_t pfcCntWr(struct file*          f,
                         char*                 buf,
                         loff_t                off,
                         size_t                len);
+static ssize_t pfcVerboseRd(struct kobject* kobj,
+						struct kobj_attribute* attr,
+						char* buf);
+static ssize_t pfcVerboseWr(struct kobject* kobj,
+						struct kobj_attribute* attr,
+						const char* buf,
+						size_t count);
+static ssize_t pfcCR4PceRd(struct kobject* kobj,
```

Comment:
This is a new method which exposes the value of CR4.PCE, so userland can check it and fail gracefully if it is zero.